gdk4/src/auto/motion_event.rs

```diff
@@ -0,0 +1,30 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files.git)
+// DO NOT EDIT
+
+use crate::Event;
+use glib::translate::*;
+use glib::StaticType;
+use std::fmt;
+
+glib::wrapper! {
+    #[doc(alias = "GdkMotionEvent")]
+    pub struct MotionEvent(Shared<ffi::GdkMotionEvent>);
+
+    match fn {
+        ref => |ptr| ffi::gdk_event_ref(ptr as *mut ffi::GdkEvent),
+        unref => |ptr| ffi::gdk_event_unref(ptr as *mut ffi::GdkEvent),
+    }
+}
+
+impl glib::StaticType for MotionEvent {
```

Comment:
Why is this not moved into `glib::wrapper!`? It can do `type_ => ...` for `Shared` types

Reply:
See the gir PR, it auto implement Value traits which is not a desired behaviour

Reply:
Right, it would need the value functions somehow then.